src/test/java/TaskListTest.java

```diff
@@ -0,0 +1,30 @@
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+    @Test
+    public void numTasksTest1(){
+        List<Task> lst = new ArrayList<>();
+        lst.add(new Task("test"));
+        TaskList testList = new TaskList(lst);
+        
+        String expected = "1 task";
+        String result = testList.numTasks();
+        assertEquals(expected, result);
+    }
+
+    @Test
+    public void numTasksTest2(){
+        List<Task> lst = new ArrayList<>();
+        for (int i = 0; i < 3; ++i) {
+            lst.add(new Task("test"));
+        }
+        TaskList testList = new TaskList(lst);
+
+        String expected = "3 tasks";
```

Comment:
Naming standards suggest that we should use names to explain and distinguish between methods and variables. I'm not very sure as to what you are trying to test just by looking at the names of the test methods. It may be better to explain what each test does in its name.